packages/devtools_app/lib/src/screens/deep_link_validation/select_project_view.dart

```diff
@@ -82,15 +82,26 @@ class _SelectProjectViewState extends State<SelectProjectView>
 
   @override
   Widget build(BuildContext context) {
-    Widget? child;
     if (_retrievingFlutterProject) {
-      child = const CenteredCircularProgressIndicator(size: _kMessageSize);
-    } else {
-      child = Text(
-        'Pick a flutter project from your local file to check all deep links status',
-        textAlign: TextAlign.center,
-        style: TextStyle(
-          color: Theme.of(context).textTheme.displayLarge!.color,
+      return Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              'Project loading',
```

Comment:
should this have a trailing ellipsis `...` to match the style of the loading message above?